src/Hosting/Hosting.sln

```diff
@@ -197,7 +191,7 @@ Global
 		{02741DAE-CF50-40C7-A717-1DC7182B737C} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
 		{87D20FC7-94AF-4FCF-A25F-221D61A1E2D9} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
 		{28899903-4A65-466C-B8C6-DA5A5CB5DF75} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
-		{006042D1-A053-4788-8D58-AB08DBB2E668} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
+		{} = {2A533A99-96D8-4A02-8990-BA8F94E70567}
```

Comment:
```suggestion
```

Reply:
I dunno if I want to force a rebuild for this change 😄

Reply:
Ah this got added somehow...

Reply:
Is VS willing to open the solution without a fix here?